bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/type/ThingTypeUtil.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.type;
+
+import java.util.List;
+
+import org.eclipse.smarthome.core.thing.ChannelUID;
+
+/**
+ *
+ * {@link ThingTypeUtil} is a helper class for working with thing types.
+ *
+ * @author Dennis Nobel - Initial contribution
+ */
+public class ThingTypeUtil {
+
+    /**
+     * Returns a channel type for the given channel UID from the thing type. The channel UID must be a channel, which is
+     * defined in a thing, which thing types matches the given thing type.
+     *
+     * @param thingType thing type
+     * @param channelUID channel UID
+     * @return channel type or null if no matching channel type could be found in the thing type
+     */
+    public static ChannelType getChannelType(ThingType thingType, ChannelUID channelUID) {
```

Comment:
Instead of introducing a new helper class, wouldn't it make sense to add this method directly on the ThingType (and remove its first parameter)?